simespresso/io_QE/tests/test_start_espresso.py

```diff
@@ -0,0 +1,24 @@
+
+import unittest
+import os
+from simespresso.io_QE import espresso_class
+
+
+class OutcomesTest(unittest.TestCase):
```

Comment:
>ok - will put all tests in test_qe.py
I see the `test_qe.py` contains all the tests.   I think the other files with the repeated testing code can be removed then.

> as for start_qe - shouldnt that be in the espresso_class, since other simphony modules will be wanting to call it ?
Where what functions are located is up to you.  I was just thinking that `start_qe` is a good candidate as it is independent of the rest of `espresso_class.py`.   so it could be left where it is, changed to a static method, or create a `start_qe` module